Bottom: 2db432b1639e1a37dd172b1505a98de5d70f26b3
Top:    872378db145a05b7d5e90666103cd3ce0882cdb3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-10 23:08:37 -0500

Refresh of fix-outline-drawing-to-only-ap

---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index b40303b..5587347 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -992,8 +992,8 @@ ps_set_layer (const char *name, int group, int empty)
     fprintf (global.f, "gsave tx ty translate 1 -1 scale 0 0 moveto ( ) show grestore newpath /ty ty ts sub def\n");
 #endif
 
-  /* If we're printing a layer other than the outline layer, and
-     we want to "print outlines", and we have an outline layer,
+  /* If we're printing a copper layer other than the outline layer,
+     and we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
       global.is_copper &&
